### PR TITLE
--delay and --line-length

### PR DESCRIPTION
**Pull request by JulienPalard**
What about giving the speed and line length as arguments? With `--delay 0 --line-length 2` I get:

### CODE REVIEW

**Comment by francofgp** [comment]
I like it even better now!.
I just found that typo, and I will merge it 💪

**Comment by francofgp** [comment]
Wow, you went all out! 💯
I like it even better now!.
I just found a tiny typo, and I will merge it 💪

File: app.py [hunk 1]
```diff
@@ -1,37 +1,51 @@
...
+import argparse
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument(
+        "--delay",
+        type=int,
+        default=1,
+        help="Drawing delay in ms (Use `0` for very fast drawings",
```
Comment (francofgp) [inline app.py]: Missing  ")" in `help` parameter.
It should be
help="Drawing delay in ms (Use `0` for very fast drawings)"

File: app.py [hunk 2]
```diff
@@ -1,37 +1,51 @@
...
+    )
+    parser.add_argument("--line-length", type=int, default=10, help="Length the lines.")
```
Comment (francofgp) [inline app.py]: I think it should be  "Length of the lines" or "Length of a line" as you wish.

**Comment by JulienPalard** [comment]
Thanks for proofreading, I fixed my errors.

**Comment by francofgp** [comment]
Merci beaucoup for your contribution!

**Comment by JulienPalard** [comment]
Avec plaisir !

**Pull request merged**